shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -313,17 +316,12 @@ @implementation FlutterKeyCallbackGuard {
 - (nonnull instancetype)initWithCallback:(FlutterAsyncKeyCallback)callback {
   self = [super init];
   if (self != nil) {
-    _callback = [callback copy];
+    _callback = callback;
```

Comment:
I can't think of why these blocks would need to be copied instead of retained.

Reply:
Yes, i remember in the old days blocks need to be copied, since it's created on the stack for performance reason and needs to be copied to the heap. In ARC this is automatic